docs/lib/storage/fragments/ios/upload.md

```diff
@@ -109,3 +109,27 @@ receiveValue: { data in
 </amplify-block>
 
 </amplify-block-switcher>
+
+## Cancel, Pause, Resume
+
+After you call `uploadFile` as above, your containing class retains a reference to the operation that is actually performing the upload.
```

Comment:
```suggestion
Calls to `uploadData` or `uploadFile` return a reference to the operation that is actually performing the upload.
```

Reply:
Same recommendation as download, except that I added uploadData for completeness and to be consistent with the download documentation.